cpp/ps0001/problem solve 1.cpp

```diff
@@ -1,19 +1,23 @@
 #include <iostream>
 using namespace std;
 int main(){
-    int first;
-    int second;
-    int third;
-    int sum;
-    cout <<"please enter first number\n";
-    cin >>first;
-    cout <<"please enter second number\n";
-    cin >>second;
-    cout <<"please enter third number\n";
-    cin >>third;
+    // prompting user for 3 numbers to add 1/2
+    short first;
+    short second;
+    short third;
+    short sum;
+    cout << "please enter first number\n";
+    cin >> first;
+    cout << "please enter second number\n";
+    cin >> second;
+    cout << "please enter third number\n";
+    cin >> third;

```

Comment:
- [ ] declared 3 variables to store user inputs with data type `unsigned short int`
- [x] used `std::cin` in a right way to prompt user for 3 numbers to add